examples/cis2-multi-royalties/src/lib.rs

```diff
@@ -0,0 +1,1481 @@
+        Amount::zero(),
+    )?;
+    Ok(())
+}
+
+/// Get the supported standards or addresses for a implementation given list of
+/// standard identifiers.
+///
+/// It rejects if:
+/// - It fails to parse the parameter.
+#[receive(
+    contract = "cis2_multi_royalties",
+    name = "supports",
+    parameter = "SupportsQueryParams",
+    return_value = "SupportsQueryResponse",
+    error = "ContractError"
+)]
+fn contract_supports<S: HasStateApi>(
+    ctx: &impl HasReceiveContext,
+    host: &impl HasHost<State<S>, StateApiType = S>,
+) -> ContractResult<SupportsQueryResponse> {
+    // Parse the parameter.
+    let params: SupportsQueryParams = ctx.parameter_cursor().get()?;
+
+    // Build the response.
+    let mut response = Vec::with_capacity(params.queries.len());
+    for std_id in params.queries {
+        if SUPPORTS_STANDARDS.contains(&std_id.as_standard_identifier()) {
+            response.push(SupportResult::Support);
+        } else {
+            response.push(host.state().have_implementors(&std_id));
+        }
+    }
+    let result = SupportsQueryResponse::from(response);
+    Ok(result)
+}
+
+/// Set the addresses for an implementation given a standard identifier and a
+/// list of contract addresses.
+///
+/// It rejects if:
+/// - Sender is not the owner of the contract instance.
+/// - It fails to parse the parameter.
+#[receive(
+    contract = "cis2_multi_royalties",
+    name = "setImplementors",
+    parameter = "SetImplementorsParams",
+    error = "ContractError",
+    mutable
+)]
+fn contract_set_implementor<S: HasStateApi>(
+    ctx: &impl HasReceiveContext,
+    host: &mut impl HasHost<State<S>, StateApiType = S>,
+) -> ContractResult<()> {
+    // Authorize the sender.
+    ensure!(ctx.sender().matches_account(&ctx.owner()), ContractError::Unauthorized);
+    // Parse the parameter.
+    let params: SetImplementorsParams = ctx.parameter_cursor().get()?;
+    // Update the implementors in the state
+    host.state_mut().set_implementors(params.id, params.implementors);
+    Ok(())
+}
+
+/// Returns the address of the payee and the amount of royalties to pay them
+///
+/// It rejects if:
+/// - The contract does not pay royalties
+/// - The return result would be 0
+/// - The token is not recognised
+#[receive(
+    contract = "cis2_multi_royalties",
+    name = "check_royalty",
+    parameter = "CheckRoyaltyParams",
+    error = "ContractError",
+    mutable
+)]
+fn contract_check_royalty<S: HasStateApi>(
+    ctx: &impl HasReceiveContext,
+    host: &mut impl HasHost<State<S>, StateApiType = S>,
+) -> ContractResult<CheckRoyaltyResult> {
+    ensure!(
+        host.state().pay_royalty,
+        concordium_cis2::Cis2Error::Custom(CustomContractError::ContractDoesNotPayRoyalties)
+    );
+
+    let params: CheckRoyaltyParams = ctx.parameter_cursor().get()?;
+    ensure!(
+        params.sale_price > 0,
+        concordium_cis2::Cis2Error::Custom(CustomContractError::NoRoyaltyPayable)
```

Comment:
```suggestion
        CustomContractError::NoRoyaltyPayable.into()
```

This appears a couple of times throughout the smart contract. You shorten the syntax by using `into()`.

Reply:
Done